docs/admin/deployment.md

```diff
@@ -452,14 +452,19 @@ storage selection.
 
 #### Discovery
 
-`dmg storage scan` can be run to query remote running `daos_server`
-processes over the management network.
-
-`daos_server storage scan` can be used to query local `daos_server` instances
-directly (scans locally-attached SSDs and Intel Persistent Memory Modules usable
-by DAOS).
-NVMe SSDs need to be made accessible first by running
-`daos_server storage prepare --nvme-only`.
+DAOS tools will discover NVMe SSDs and Persistent Memory Modules using the storage scan commands.
+
+`dmg storage scan` can be run to query remote running `daos_server` processes over the management
+network.
+
+`daos_server storage scan` can be used to query storage on local hosts directly.
+
+NVMe SSDs need to be made accessible first by running `daos_server storage prepare --nvme-only`.
+The default way for DAOS to access NVMe storage is through SPDK via the VFIO user-space driver.
+To use an alternative driver with SPDK, set `--disable-vfio` in the storage prepare command to
+fallback to using UIO user-space driver with SPDK instead.
+If IOMMU and VFIO are not enabled in the BIOS, the alternative driver will be used by default.
```

Comment:
New feature ?
Until now, I had to explicitly put disable_vfio in the config file (from what I remember of my last install of DAOS).

Reply:
SPDK will not bind devices to the vfio-pci driver unless it is available, therefore if unavailable (because IOMMU has not been enabled in the BIOS) SPDK should instead bind the devices to uio-generic driver. Do you remember why it would fail if you did not explicitly disable VFIO? Do you have an environment where you could verify this?

Reply:
I think it's best to encourage people to be explicit when not wanting to use VFIO so I have removed the comment about default behaviour.

Reply:
done

Reply:
I am also probably makeing some confusion with the fact that you have to be root for using uio.
However, as you said, I prefer to explicitly define and thus to not forget that I am using DAOS in a degraded mode.